models/lpjguess/R/model2netcdf.LPJGUESS.R

```diff
@@ -63,6 +63,10 @@ model2netcdf.LPJGUESS <- function(outdir, sitelat, sitelon, start_date, end_date
   ### Loop over years in LPJ-GUESS output to create separate netCDF outputs
   for (y in years){
     
+    if (file.exists(file.path(outdir, paste(y,"nc", sep=".")))) {
+      next
+    }
```

Comment:
Do you need an else statement there? Maybe print no file or something so it shows up in the logfile.txt? Not a big deal though.